com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -121,18 +401,29 @@ class LightDefinitions
         public static uint s_ScreenSpaceColorShadowFlag = 0x100;
         public static uint s_InvalidScreenSpaceShadow = 0xff;
         public static uint s_ScreenSpaceShadowIndexMask = 0xff;
+
+        // Binned lighting
+        // For performance reasons, keep all sizes in powers of 2.
+        public static int s_CoarseTileEntryLimit = 64; // Per category; before pruning, so the number is lower in practice; the number could be made (almost always) exact at the cost of making the 'FillCoarseTiles' pass more complicated
+        public static int s_FineTileEntryLimit   = 16; // Per category; after pruning, so the number is exact
+        public static int s_CoarseTileSize       = 64;
+        public static int s_FineTileSize         = 8;
+        public static int s_zBinCount            = 8192;
+        public static int s_MaxReflectionProbesPerPixel = 4;
```

Comment:
>s_MaxReflectionProbesPerPixel = 4
we may need to revisit that. Even if this is what we should have, I expect many unity project to have more due to "users not caring"

Reply:
Yeah, this was done very quickly...